edb/server/metrics.py

```diff
@@ -200,6 +200,12 @@
     labels=('tenant', 'extension'),
 )
 
+feature_used = registry.new_labeled_gauge(
+    'feature_used_branch_count_current',
+    'How many branches a schema feature is used by.',
+    labels=('tenant', 'feature'),
+)
```

Comment:
@zackelan please confirm we can plot this later

Reply:
yes, this fits well into the Prometheus data model, so shouldn't pose any problems with aggregating it after deployment